### ISSUE TITLE
New submissions for Wednesday, 21 August 2024 (showing 44 of 44 entries )

### ISSUE THREAD

**Issue by terrryahn78** [terrryahn78/daily_arxiv_noti#7]
## Room-temperature cavity exciton-polariton condensation in perovskite quantum dots
- **Keywords:** ['semiconductor quantum dot']
 - **Authors:** Ioannis Georgakilas, David Tiede, Darius Urbonas, Clara Bujalance, Laura Caliò, Rafał Mirek, Virginia Oddi, Rui Tao, Dmitry N. Dirin, Gabriele Rainò, Simon C. Boehme, Juan F. Galisteo-López, Rainer F. Mahrt, Maksym V. Kovalenko, Hernán Miguez, Thilo Stöferle
 - **Subjects:** Subjects:
Optics (physics.optics); Mesoscale and Nanoscale Physics (cond-mat.mes-hall); Other Condensed Matter (cond-mat.other)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2408.10667
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2408.10667
 - **Abstract**
 The exploitation of the strong light-matter coupling regime and exciton-polariton condensates has emerged as a compelling approach to introduce strong interactions and nonlinearities into numerous photonic applications, ranging from low-threshold topological lasers to ultrafast all-optical logic devices. The use of colloidal semiconductor quantum dots with strong three-dimensional confinement as the active material in these microcavities would be highly advantageous due to their versatile structural and compositional tunability and wet-chemical processability, as well as potentially enhanced, confinement-induced polaritonic interactions. Yet, to date, cavity exciton-polariton condensation has neither been achieved with epitaxial nor with colloidal quantum dots. Here, we demonstrate room-temperature polariton condensation in a thin film of monodisperse, colloidal CsPbBr$_3$ quantum dots placed in a tunable optical resonator with a Gaussian-shaped deformation serving as wavelength-scale potential well for the polaritons. The onset of polariton condensation under pulsed optical excitation is manifested in emission by its characteristic superlinear intensity dependence, reduced linewidth, blueshift, and extended temporal coherence. Our results, based on this highly engineerable class of perovskite materials with unique optical properties, pave the way for the development of polaritonic devices for ultrabright coherent light sources and photonic information processing.
## Demonstration of Hardware Efficient Photonic Variational Quantum Algorithm
- **Keywords:** ['hybrid quantum']
 - **Authors:** Iris Agresti, Koushik Paul, Peter Schiansky, Simon Steiner, Zhengao Yin, Ciro Pentangelo, Simone Piacentini, Andrea Crespi, Yue Ban, Francesco Ceccarelli, Roberto Osellame, Xi Chen, Philip Walther
 - **Subjects:** Subjects:
Quantum Physics (quant-ph)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2408.10339
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2408.10339
 - **Abstract**
 Quantum computing has brought a paradigm change in computer science, where non-classical technologies have promised to outperform their classical counterpart. Such an advantage was only demonstrated for tasks without practical applications, still out of reach for the state-of-art quantum technologies. In this context, a promising strategy to find practical use of quantum computers is to exploit hybrid quantum-classical models, where a quantum device estimates a hard-to-compute quantity, while a classical optimizer trains the parameters of the model. In this work, we demonstrate that single photons and linear optical networks are sufficient for implementing Variational Quantum Algorithms, when the problem specification, or ansatz, is tailored to this specific platform. We show this by a proof-of-principle demonstration of a variational approach to tackle an instance of a factorization task, whose solution is encoded in the ground state of a suitable Hamiltonian. This work which combines Variational Quantum Algorithms with hardware efficient ansatzes for linear-optics networks showcases a promising pathway towards practical applications for photonic quantum platforms.
## Robust quantum dots charge autotuning using neural networks uncertainty
- **Keywords:** ['semiconductor spin qubit']
 - **Authors:** Victor Yon, Bastien Galaup, Claude Rohrbacher, Joffrey Rivard, Clément Godfrin, Ruoyu Li, Stefan Kubicek, Kristiaan De Greve, Louis Gaudreau, Eva Dupont-Ferrier, Yann Beilliard, Roger G. Melko, Dominique Drouin
 - **Subjects:** Subjects:
Quantum Physics (quant-ph); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/arXiv:2406.05175
 - **Pdf link:** https://arxiv.org/pdf/arXiv:2406.05175
 - **Abstract**
 This study presents a machine-learning-based procedure to automate the charge tuning of semiconductor spin qubits with minimal human intervention, addressing one of the significant challenges in scaling up quantum dot technologies. This method exploits artificial neural networks to identify noisy transition lines in stability diagrams, guiding a robust exploration strategy leveraging neural networks' uncertainty estimations. Tested across three distinct offline experimental datasets representing different single quantum dot technologies, the approach achieves over 99% tuning success rate in optimal cases, where more than 10% of the success is directly attributable to uncertainty exploitation. The challenging constraints of small training sets containing high diagram-to-diagram variability allowed us to evaluate the capabilities and limits of the proposed procedure.
